dll/win32/shell32/CCopyMoveToMenu.h

```diff
@@ -109,3 +109,35 @@ class CMoveToMenu
     UINT GetFileOp() const override { return FO_MOVE; }
     LPCSTR GetVerb() const override { return "moveto"; }
 };
+
+class CCopyAsPathMenu 
```

Comment:
```suggestion
class CCopyAsPathMenu
```
There is EOL-whitespace introduced in line 113. Possibly otherwhere too, use NPP to sledgehammer all of them out!

Reply:
I do have whitespace visible, just missed that one, thanks.